tests/foreman/api/test_capsulecontent.py

```diff
@@ -1595,3 +1596,56 @@ def test_positive_content_counts_blank_update(
         assert (
             counts is None or len(counts['content_view_versions']) == 0
         ), f"No content counts expected, but got:\n{counts['content_view_versions']}."
+
+    def test_positive_reclaim_space(
+        self,
+        target_sat,
+        module_capsule_configured,
+    ):
+        """Verify the reclaim_space endpoint spawns the Reclaim space task
+        and apidoc references the endpoint correctly.
+
+        :id: eb16ed53-0489-4bb9-a0da-8d857a1c7d06
+
+        :setup:
+            1. A registered external Capsule.
+
+        :steps:
+            1. Trigger the reclaim space task via API, check it succeeds.
+            2. Check the apidoc references the correct endpoint.
+
+        :expectedresults:
+            1. Reclaim_space endpoint spawns the Reclaim space task and it succeeds.
+            2. Apidoc references the correct endpoint.
+
+        :CaseImportance: Medium
+
+        :BZ: 2218179
+
+        :customerscenario: true
+        """
+        # Trigger the reclaim space task via API, check it succeeds
+        task = module_capsule_configured.nailgun_capsule.content_reclaim_space()
+        assert task, 'No task was created for reclaim space.'
+        assert (
+            'Actions::Pulp3::CapsuleContent::ReclaimSpace' in task['label']
+        ), 'Unexpected task triggered'
+        assert 'success' in task['result'], 'Reclaim task did not succeed'
+
+        # Check the apidoc references the correct endpoint
+        response = client.get(
+            f'{target_sat.url}/apidoc/v2/capsule_content/reclaim_space.en.html',
+            auth=get_credentials(),
+            verify=False,
+        )
+        assert (
+            response.status_code == 200
+        ), f'{response.request.path} returned HTTP{response.status_code}'
+
+        ep_path = target_sat.api.Capsule(id=module_capsule_configured.nailgun_capsule.id).path(
+            which='content_reclaim_space'
+        )
+        ep_path = urlparse(ep_path).path
+        ep_path = ep_path.replace(str(module_capsule_configured.nailgun_capsule.id), ':id')
+
+        assert f'POST {ep_path}' in response.text, 'Reclaim_space endpoint path missing in apidoc'
```

Comment:
This is setting a new standard of API doc testing. I would suggest to choose from the following two ways of doing it instead of what you implemented.
* use https://github.com/Apipie/apypie to fetch the data from the API doc (JSON) and perform assertions on the data it returns (reads from Satellite API). This is the most elegant way of doing things, I would say. :smiley: 
* do `apypie`'s work manually and fetch the data from the JSON API doc. Dummy example below.

```python
import requests
res = requests.get('https://satellite.com/apidoc/v2.en.json', verify=False)
res_json = res.json()
reclaim_space_ep = None
for method in res_json['docs']['resources']['capsule_content']['methods']:
    if method['name'] == 'reclaim_space':
        reclaim_space_ep = method
assert len(reclaim_space_ep['apis']) == 1
assert reclaim_space_ep['apis'][0]['http_method'] == 'POST'
assert reclaim_space_ep['apis'][0]['api_url'] == '/katello/api/capsules/:id/content/reclaim_space'
```

If there are going to be more test cases testing the API doc, we should think of introducing some kind of helper for it. I am not sure how would it look ATM, but such helpers should reduce code repetition.